docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.de.md

```diff
@@ -535,7 +551,22 @@ driver.get("http://www.google.com")
 driver.quit()
   {{< / code-panel >}}
   {{< code-panel language="csharp" >}}
-// Please raise a PR to add code sample
+using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.IE;
+
+namespace ieTest {
+    class Program {
+        static void Main(string[] args) {
+            InternetExplorerOptions options = new InternetExplorerOptions();
+            options.ForceCreateProcessApi = true;
+            IWebDriver driver = new InternetExplorerDriver(options);
+            driver.Url = "https://google.com/ncr";
```

Comment:
HI @kjayachandra2000 ,

Code sample should only explain the initialisation of driver options. i think you can remove the next lines after this step. 

Also, can you restrict line spacing to 2 spaces instead of 4. there is lot of free space to the left which we want to use as much as possible.

Same for all code samples.

Reply:
Hi @Harsha509 ,
I have followed other code snippets for these options, check the link FYI https://www.selenium.dev/documentation/en/driver_idiosyncrasies/driver_specific_capabilities/#ie-command-line-options
But anyways I have done changes as mentioned by you.
![image](https://user-images.githubusercontent.com/11714052/83240626-b87e0400-a1aa-11ea-9dda-c7d2456afc59.png)